src/metaOutput.cxx

```diff
@@ -304,7 +304,7 @@ METAIO_STL::string MetaOutput::
 GetUsername()
 {
 #if defined (_WIN32) && !defined(__CYGWIN__)
-    static char buf[1024];
+    char buf[1024];
```

Comment:
This char array cannot be returned.   Its memory is freed after the call completes.  See return in line 311.